imageroot/actions/get-configuration/10get

```diff
@@ -5,13 +5,66 @@
 # SPDX-License-Identifier: GPL-3.0-or-later
 #
 
+from datetime import datetime, timezone
+
 import os
 import json
 import sys
+import subprocess
+
+# Cloud Log Manager
+
+cloud_log_manager = {}
+
+with subprocess.Popen(['systemctl', '--user', 'is-active', 'cloud-log-manager-forwarder'], stdout=subprocess.PIPE, stderr=sys.stderr) as pipe:
+    response = pipe.stdout.readline().decode('utf-8').replace('\n', '')
+    if 'inactive' in response:
+        cloud_log_manager["status"] = 'inactive'
+    elif 'active' in response:
+        cloud_log_manager["status"] = 'active'
+    elif 'failed' in response:
+        cloud_log_manager["status"] = 'failed'
```

Comment:
You can make this way easier using `subprocess.run`, here's an example: 
```python
response = subprocess.run(['systemctl', '--user', 'is-active', 'cloud-log-manager-forwarder'], capture_output=True, text=True)
```

Then you can check the output with:

```python
if 'inactive' in response.stdout':
   ...
```

Worry not about converting text, the `text` set as True will do that for you, you can even omit the `with`!